client/tests/phpunit/Service/Audit/AuditEventsTest.php

```diff
@@ -0,0 +1,43 @@
+<?php declare(strict_types=1);
+
+namespace AppBundle\Service\Audit;
+
+use PHPUnit\Framework\TestCase;
+
+class AuditEventsTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function clientDischarged(): void
+    {
+        $expected = [
+            'event' => AuditEvents::CLIENT_DISCHARGED,
+            'trigger' => 'ADMIN_BUTTON',
+            'case_number' => '19348522',
+            'type' => 'audit'
+        ];
+
+        $actual =  (new AuditEvents())->clientDischarged('ADMIN_BUTTON', '19348522');
```

Comment:
I know its not really what this is testing, but would the event type be different here? Looks like we have a couple of different event types for discharges - maybe `CLIENT_DISCHARGED `?

Reply:
Ah yes, technically it would be 'CSV_UPLOAD', which represents the trigger/source of the event. But for the purpose of the test it could as well be 'FOO', so long as what goes in also comes out.